show_all_imagestreams.py

```diff
@@ -57,8 +53,9 @@ def show_all_imagestreams(self):
             print(f"Tags in the image stream {f}:")
             for tag in json_dict["spec"]["tags"]:
                 print(f"- {tag['name']} -> {tag['from']['name']}")
+        return 0
 
 
 if __name__ == "__main__":
     isc = ShowAllImageStreams()
-    isc.show_all_imagestreams()
+    sys.exit(isc.show_all_imagestreams())
```

Comment:
Why do we need exiting the script with sys.exit?

Reply:
Good point. We do not need it. I have used in script check_imagestreams.py.

Reply:
Fixed by https://github.com/sclorg/container-common-scripts/pull/372/commits/43d7e4f42d6655ba35492b90cb5dd30dc085d449